build/zlib-1.2.8/contrib/vstudio/vc14/zlibstat.vcxproj

```diff
@@ -171,38 +171,11 @@
     <CodeAnalysisRules Condition="'$(Configuration)|$(Platform)'=='Release|x64'" />
     <CodeAnalysisRuleAssemblies Condition="'$(Configuration)|$(Platform)'=='Release|x64'" />
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='ReleaseWithoutAsm|Win32'">
```

Comment:
Ah right, I'm just saying that when we have to merge our changes with the latest version it'll be more diffs to look through. If we're not using ReleaseWithoutAsm I think it's fine to leave it in there untouched, but maybe you and @Pegasus-RPG don't agree.